src/modules/app/containers/__tests__/__snapshots__/App.spec.js.snap

```diff
@@ -70,7 +70,298 @@ exports[`App should match snapshot 1`] = `
     }
   >
     <Connect(I18nProvider)>
-      <RouterFragment />
+      <RouterFragment
```

Comment:
I don't like this snapshot. Way too long and too much information.

Reply:
@maxammann just changed it so that the pr looks a lot bigger than it is :P 300 lines extra 👍

Reply:
:P would need to replace this with a normal test then.